virttest/qemu_devices/qcontainer.py

```diff
@@ -2241,6 +2241,7 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
         :param nr: the nr(port) of the parent bus(optional, virtio only)
         :param reg: reg option of isa-serial(optional)
         :param bus_extra_params: extra virtio_serial_pci params
+        :param reuse_bus: if reuse existing virtio-serial-pci bus
```

Comment:
Better highlight it is for virtserialport only.
Others LGTM

Reply:
thanks and update.